watertap/examples/flowsheets/RO_with_energy_recovery/monte_carlo_sampling_RO_ERD.py

```diff
@@ -61,9 +72,13 @@ def run_parameter_sweep(results_file, seed=None, use_LHS=False):
     # Run the parameter sweep study using num_samples randomly drawn from the above range
     num_samples = 10
 
+    # Filename of the h5 file that we want to write but without the extension
```

Comment:
It should be easy to support both with and without the extension.

Reply:
Incorporated the ability to handle without or without the extension using the function here 
https://github.com/watertap-org/watertap/blob/6899183f16af7c5e9e195a4c8de6b53ff52acbc3/watertap/tools/parameter_sweep.py#L153-L158